package.json

```diff
@@ -43,9 +43,9 @@
     "tslint": "tslint -c tslint.json -p tsconfig.json",
     "test": "nyc mocha -r ts-node/register --recursive \"tests/**/*.spec.ts\" --exit",
     "test-js": "mocha -r ts-node/register --recursive \"lib/tests/**/*.spec.js\"",
-    "napi": "prebuildify --napi --strip --verbose --target=node@10.12.0 --target=electron@3.1.6",
-    "napi-64": "prebuildify --napi --strip --verbose --target=node@10.12.0 --target=electron@3.1.6 --arch=x64",
-    "napi-32": "prebuildify --napi --strip --verbose --target=node@10.12.0 --target=electron@3.1.6 --arch=ia32",
+    "napi": "prebuildify --napi --strip --verbose --target=node@12.4.0 --target=electron@3.1.6",
```

Comment:
I think we need to keep node 10 for a while but should add node 12

Reply:
Good point